exporter/src/main/java/io/zeebe/exporters/kafka/record/RecordHandler.java

```diff
@@ -43,10 +44,13 @@ public RecordHandler(final @NonNull RecordsConfig configuration) {
    * @param record the record to transform
    * @return the transformed record
    */
-  public @NonNull ProducerRecord<RecordId, Record> transform(final @NonNull Record record) {
+  public @NonNull ProducerRecord<RecordId, byte[]> transform(final @NonNull Record record) {
     final RecordConfig config = getRecordConfig(record);
+    final byte[] serializedRecord = record.toJson().getBytes(StandardCharsets.UTF_8);
```

Comment:
How come you are doing serialization in here ?  Why not leave this method as as returning `ProducerRecord<RecordId, Record>`.  So that this PR can remain compatible with changes in https://github.com/zeebe-io/zeebe-kafka-exporter/pull/37 which supports switching out the serde implementation to support protobuf

Reply:
Good point; I was originally doing it to avoid serializing twice, and as a way to estimate how much memory is currently buffered. We could still do this by simply calling the right serializer ourselves though, no?

Reply:
Yep, agreed